examples/Client/DistributedLock/Program.cs

```diff
@@ -21,7 +21,7 @@ public static void Main(string[] args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
                     webBuilder.UseStartup<Startup>()
-                        .UseUrls($"http://localhost:{Environment.GetEnvironmentVariable("APP_PORT")}");
+                        .UseUrls($"http://localhost:{Environment.GetEnvironmentVariable("APP_PORT") ?? "22222"}");
```

Comment:
nit: Instead of directly hardcoding, do you want to define this in a variable like `DEFAULT_APP_PORT` for better readability ?

Reply:
Yup! Makes sense!